src/nfa/operations.cc

```diff
@@ -632,24 +632,21 @@ std::pair<Run, bool> mata::nfa::Nfa::get_word_for_path(const Run& run) const {
 }
 
 //TODO: this is not efficient
-bool mata::nfa::Nfa::is_in_lang(const Run& run) const {
+bool mata::nfa::Nfa::is_in_lang(const Run& run, const bool use_epsilon, const bool match_prfx) const {
```

Comment:
```suggestion
bool mata::nfa::Nfa::is_in_lang(const Run& run, const bool use_epsilon, const bool match_prefix) const {
```
I would use the longer names.

Reply:
I agree. I took the name `prfx` from the old implementation.